eats/raleigh-nc.md

```diff
@@ -0,0 +1,4 @@
+### Raleigh, NC Great Places to Eat
+
+* Aviator's in Fuquay
+* The Pit for BBQ (North Carolina style)
```

Comment:
@githubteacher, did you want to add a link to The Pit's Yelp page? http://www.yelp.com/biz/the-pit-raleigh